spec/requests/dashboards_spec.rb

```diff
@@ -8,7 +8,7 @@
 
   before { sign_out }
 
-  context 'when user has no workgroup' do
+  context 'when user has no deposits' do
```

Comment:
Was the prior test bogus?

Reply:
Yes, it comes from when we thought there was a group that all users had to be part of.